py/README.md

```diff
@@ -0,0 +1,26 @@
+# Torchcraft
+
+This the alpha version of pytorchcraft v0.0.1, and current is parity with the
```

Comment:
arg what number should I give it? I was going to just bump it up to parity once we use it for a few weeks.